lib/git_worktree.rb

```diff
@@ -139,7 +160,7 @@ def mv_dir(old_dir, new_dir)
 
   def fetch_and_merge
     fetch
-    commit = @repo.ref("refs/remotes/#{@remote_name}/master").target
+    commit = @repo.ref("refs/remotes/#{@remote_name}/#{@current_branch}").target
```

Comment:
@Fryguy @bdunne 
Should we make the commit_sha a private variable and only allow them to change to a branch or a tag.